frontend/lib/src/components/widgets/ChatInput/ChatInput.tsx

```diff
@@ -61,7 +61,10 @@ const isEnterKeyPressed = (
   // https://bugs.chromium.org/p/chromium/issues/detail?id=79407
 
   const { keyCode, key } = event
-  return key === "Enter" || keyCode === 13 || keyCode === 10
+  return (
+      (key === "Enter" || keyCode === 13 || keyCode === 10) &&
```

Comment:
Ah, it also looks like the linter is complaining about the size of the indents on these lines (we use 2-space indents in all of our typescript files)